src/sections/hero/index.jsx

```diff
@@ -15,6 +28,11 @@ const Hero = () => {
         seu estado durante o surto de COVID-19. Veja a evolução da pandemia na
         sua área de residência.
       </Typography>
+      <Button href={data.site.siteMetadata.appSiteUrl} type="cta">
+        Começar a usar
```

Comment:
You can use `typograhy` as a child of `Button`, and remove font sizes from button css file

Reply:
Updated!